examples/example_golden_files/complex_resource/apply.json

```diff
@@ -37,11 +37,9 @@
   },
   {
     "@level": "info",
-    "@message": "tfcoremock_complex_resource.complex_resource: Creation complete after 0s [id=d199d8ea-e8f8-4fb0-8276-3567a74d3db8]",
```

Comment:
I can see that these would now be removed by running the tool's `update` command, so it makes sense to update them in this PR 👍🏻